src/range/range_formatter.py

```diff
@@ -17,12 +17,10 @@ def summary(valobj, dict):
                 return str(start)
             return "(folly::StringPiece) " + summary
         else:
-            return "folly::Range size=" + str(
-                SyntheticFormatter.get_range_length(valobj)
-            )
+            return f"(folly::Range) <{SyntheticFormatter.get_item_type(valobj)}> [{SyntheticFormatter.get_range_length(valobj)}]"
```

Comment:
This could be even better. For example, when you print a list<int> you get

```
(std::list<int, std::allocator<int> >) $1 = size=3
```

so it'd be nicer if you print simply `size = <size>`

to have consistence with the other formatters

Reply:
Similarly, for the StringPiece summary, you can use the same output that you could get from printing a string
```
(std::string) $2 = "sdfsdfsdf"
```
that means that you don't need to print "(folly::StringPiece) "